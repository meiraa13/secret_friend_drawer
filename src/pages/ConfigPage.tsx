import { Footer } from "../components/Footer";
import { Form } from "../components/Form";
import { ParticipantList } from "../components/ParticipantList";

export function ConfigPage(){

    return(
        <>
            <Form />
            <ParticipantList />
            <Footer />
        </>


    )
}