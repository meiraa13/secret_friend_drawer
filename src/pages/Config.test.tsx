import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import { ConfigPage } from "./ConfigPage";

const mockNavigation = jest.fn()

jest.mock('react-router-dom',()=>{
    return {
        useNavigate:() => mockNavigation
    }
})
describe('config page',()=>{
   
    test('must be rendered',()=>{
        const { container } = render(
            <RecoilRoot>
                <ConfigPage />
            </RecoilRoot>
        )

        expect(container).toMatchSnapshot()

    })
})