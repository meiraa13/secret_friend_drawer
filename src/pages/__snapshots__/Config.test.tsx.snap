// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config page must be rendered 1`] = `
<div>
  <form>
    <input
      placeholder="Insira o nome do participante"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar Brincadeira
    </button>
  </footer>
</div>
`;
